pyV2DL3/script/v2dl3_for_vegas.py

```diff
@@ -202,26 +213,94 @@ def cli(
             fname_base = os.path.splitext(os.path.basename(st5_str))[0]
             datasource = loadROOTFiles(st5_str, ea_str, "VEGAS",
                                        event_classes=event_classes,
+                                       save_msw_msl=save_msw_msl,
                                        )
 
             datasource.set_irfs_to_store(irfs_to_store)
             with cpp_print_context(verbose=verbose):
                 datasource.fill_data()
-            hdulist = genHDUlist(datasource, save_multiplicity=save_multiplicity)
-            if filename_to_obsid:
-                logging.info(
-                    f"Overwriting OBS_ID={hdulist[1].header['OBS_ID']} with OBS_ID={fname_base}"
-                )
-                hdulist[1].header["OBS_ID"] = fname_base
-            hdulist.writeto(f"{output}/{fname_base}.fits", overwrite=True)
-            flist.append(f"{output}/{fname_base}.fits")
-            # Generate hdu obs index file
+
+            # Prepare output paths
+            output_path = f"{output}/{fname_base}"
```

Comment:
Super minor comment: `os.path.join()` provides a save way of joining paths and files and is sometimes preferred. I am just mention it, probably more a matter of style.